pt_miniscreen/components/confirmation_page.py

```diff
@@ -79,7 +79,7 @@ def perform_action(self):
             self.on_confirm
         ):
             self.on_confirm()
-            elements_to_pop = 3
+            elements_to_pop = 2
```

Comment:
a comment to explain this magic number would be good. eg # pop 2 element to get back to top of Projects menu - the project types list and (sometimes?) the project usernames list